vassal-app/src/main/java/VASSAL/build/module/documentation/AboutScreen.java

```diff
@@ -79,6 +78,18 @@ public void launch() {
       Resources.getString("AboutScreen.module_version",  //$NON-NLS-1$
         g.getLocalizedGameName(), g.getGameVersion()));
 
+
+    final String o1 = g.getModuleOther1();
+    if ((o1 != null) && !o1.isEmpty()) {
+      sb.append(" - ");
+      sb.append(o1);
+    }
+    final String o2 = g.getModuleOther2();
+    if ((o2 != null) && !o2.isEmpty()) {
+      sb.append(" - ");
+      sb.append(o1);
```

Comment:
`o1` should be `o2` on line 90.

Reply:
I think I got that in the second commit?

Reply:
Oops, didn't see the 2nd commit.  All good, then.